Package.swift

```diff
@@ -30,14 +30,12 @@ let package = Package(
         "AirbnbSwiftFormatTool",
         .product(name: "swiftformat", package: "SwiftFormat"),
         .product(name: "swiftlint", package: "SwiftLint"),
-      ],
-      path: "AirbnbSwiftFormatPlugin"),
+      ]),
     .executableTarget(
       name: "AirbnbSwiftFormatTool",
       dependencies: [
         .product(name: "ArgumentParser", package: "swift-argument-parser"),
       ],
-      path: "AirbnbSwiftFormatTool",
```

Comment:
We can remove this path by putting the source in a folder under the default Sources directory.